src/hclib-runtime.c

```diff
@@ -354,14 +355,15 @@ static inline void check_out_finish(finish_t *finish) {
         // hc_atomic_dec returns true when finish->counter goes to zero
         if (hc_atomic_dec(&(finish->counter))) {
 #if HCLIB_LITECTX_STRATEGY
```

Comment:
Last time when I tried "not using" the HCLIB_LITECTX_STRATEGY, small changes were required in the runtime to fix the alternate code path.